lib/auth/auth_with_roles_test.go

```diff
@@ -2881,7 +2881,7 @@ func TestListResources_KindKubernetesCluster(t *testing.T) {
 	srv, err := NewTestAuthServer(TestAuthServerConfig{Dir: t.TempDir()})
 	require.NoError(t, err)
 
-	authContext, err := srv.Authorizer.Authorize(context.WithValue(ctx, ContextUser, TestBuiltin(types.RoleProxy).I))
+	authContext, err := srv.Authorizer.Authorize(context.WithValue(ctx, ContextUser, TestBuiltin(types.RoleAdmin).I))
```

Comment:
I read the PR description, but why is this test failing with "access denied"? Doesn't this suggest a bug somewhere if we need to increase the privileges in order to make that test pass?